src/components/collapsible.tsx

```diff
@@ -52,9 +55,17 @@ const Collapsible: FC<Props> = ({
         className={classNames("flex items-center cursor-pointer", {
           "transition duration-200 hover:opacity-60": opacityOnHover,
         })}
-        onClick={() => {
-          onChange && onChange(!isCollapsed)
-          setIsCollapsed(!isCollapsed)
+        onClick={(e) => {
+          if (
+            !hasEventBubbledThroughTag({
+              tagName: "a",
+              target: e.target as HTMLElement,
+              currentTarget: e.currentTarget,
```

Comment:
any chance target/currentTarget could be undefined?

Reply:
I don't think so, each onClick event should have a target and a currentTarget